dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.35/V2_35_33__Add_Deferred_Constraint_to_Interpretation_Comments_Table.sql

```diff
@@ -0,0 +1,6 @@
+-- Checking unique constraint only in the end of the transaction.
+-- DHIS2-8256: Deleting interpretation comments does not always work
+
+ALTER TABLE interpretation_comments DROP CONSTRAINT IF EXISTS interpretation_comments_interpretationcommentid_key;
+
+ALTER TABLE interpretation_comments ADD CONSTRAINT interpretation_comments_interpretationcommentid_key UNIQUE (interpretationcommentid) DEFERRABLE INITIALLY DEFERRED;
```

Comment:
Just checking, the `DEFERRABLE INITIALLY DEFERRED;`, is there a reason we want this on this particular constraint?

Reply:
Hey @larshelge , you can find the reason in my original PR here:
https://github.com/dhis2/dhis2-core/pull/7563